crates/solidity/inputs/language/bindings/rules.msgb

```diff
@@ -2059,6 +2086,10 @@ inherit .star_extension
   attr (def) node_definition = @item
   attr (def) definiens_node = @item
 
+  if (version-matches ">= 0.6.8") {
+    edge def -> @enum.value_ref_typeof
+  }
```

Comment:
This should not be version restricted. I tried in Remix and binding enum members work for all our supported versions (eg. >= 0.4.11).